src/main/ts/utils/PathUtils.ts

```diff
@@ -1,8 +1,12 @@
-export const isPathInMain = (filePath: string) =>
-  filePath.startsWith('src/main/') || filePath.includes('/src/main/') || filePath.includes('/src/core/main/');
+// Replace `\` with `/` to handle Windows paths
+export const normalizeFilePath = (filePath: string) =>
+  filePath.replace(/\\/g, '/');
 
-export const isPathInTest = (filePath: string) =>
-  filePath.startsWith('src/test/') || filePath.includes('/src/test/') || (/src\/[\w\/]+\/test\//).test(filePath);
+export const isPathInMain = (normalizedFilePath: string) =>
+  normalizedFilePath.startsWith('src/main/') || normalizedFilePath.includes('/src/main/') || normalizedFilePath.includes('/src/core/main/');
```

Comment:
Could probably write these path related lints to use the path library with:
https://nodejs.org/api/path.html#pathparsepath
https://nodejs.org/api/path.html#pathsep

Still I admit we're unlikely to encounter any edge-cases which would require that.

Reply:
Yeah I did consider that, the problem there is that we couldn't test it unless we run on a Windows device which I was trying to avoid.